modelmapper/misc.py

```diff
@@ -19,6 +21,27 @@
 CHUNK_SIZE = 2048  # The chunk needs to be big enough that covers a couple of rows of data.
 
 
+valid_chars_for_string = set(string.ascii_letters.lower())
+valid_chars_for_integer = set(string.digits)
+
+MAX_DATE_INTEGER = 30001230125959  # year 3000, December 30 12:59:59
```

Comment:
Is there no constant for this? This is fine but I feel like there must be some datetime constant for this.

Reply:
This is an arbitrary number I came up with. All it does is to check if a number below this can be a datetime value. I don't expect this code to be run beyond year 3000.